src/FM.LiveSwitch.Hammer/ScanTest.cs

```diff
@@ -219,7 +219,36 @@ private async Task<bool> MediaServerWouldBeOverCapacity(string mediaServerId)
 
         private async Task<MediaServerInfo[]> GetMediaServers()
         {
-            var responseJson = await _HttpClient.GetStringAsync("v1/mediaservers").ConfigureAwait(false);
+            string responseJson = string.Empty;
+
+            var count = 0;
+            var delay = 100;
+            while(true)//we either break out of the loop, or throw an exception
+            {
+                try
+                {
+                    responseJson = await _HttpClient.GetStringAsync("v1/mediaservers").ConfigureAwait(false);
+                    break;
+                }
+                catch(System.Net.Http.HttpRequestException ex)
+                {
+                    if(count <= 5)
```

Comment:
formatting - does VS not auto-format this for you?

Reply:
I'm using vscode - I'll need to tweak the formatting rules to add whitespace